blog/_posts/2023-01-06-advent-of-code-recap.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jamie Thompson, Scala Center
+title: Thank you for taking part in Advent of Code 2022!
+description: A recap of how the Scala Community participated in the recent 2022 Advent of Code challenge
+---
+
+Happy New Year everyone! We would like to take some time in this blog post to thank everyone from the Scala community and beyond who took part in the Advent of Code challenge this year! We are also thrilled to see all the related activity in our [Discord channel][aoc-channel] on the [Scala Discord][discord] and [Scala Advent of Code website][sc-advent-of-code].
+
+*To recap, [Advent of Code](https://adventofcode.com/), started by [Eric Wastl](http://was.tl/), is a yearly event providing daily programming puzzles between December 1st and December 25th.*
+
+If you read on you will see how the Scala community participated in the challenge, and also see how to help us write articles on solving each puzzle in a **simple Scala** coding style (see Martin Odersky’s keynote talk [Simply Scala][simply-scala]).
+
+## The Scala communities’ engagement
+
+This year it was great to see over 40+ of you in the [advent-of-code channel][aoc-channel] of the [Scala Discord server][discord]! You participated daily, shared and discussed solutions, and challenged each other to find more performant or concise solutions.
+
+We would also like to thank everyone who contributed your solutions to be displayed on our refreshed [Scala Advent of Code 2022 website][sc-advent-of-code]! This year, we had 164 contributed solutions from the community, and over 5000 visitors to the website! (compared to 800 visitors last year) We’re especially thankful for those of you who additionally volunteered to write **whole articles** to explain your solution! (see [Day 10][aoc-day-10], [Day 18][aoc-day-18], and [Day 24][aoc-day-24])!
+
+![](/resources/img/blog/aoc-2022-recap/daily-solution-count.svg)
+![](/resources/img/blog/aoc-2022-recap/daily-visitor-count.svg)
```

Comment:
Did you check that the figures render correctly?

Reply:
I tried on my own computer